components/stepper/themes/express.css

```diff
@@ -38,7 +38,6 @@
 		/** Invalid **/
 		--spectrum-stepper-border-color-invalid: transparent;
 		--spectrum-stepper-border-color-focus-invalid: transparent;
-		--spectrum-stepper-border-color-focus-hover-invalid: transparent;
```

Comment:
TODO: this was looking like a bug on focus+hover. double check if this change is necessary any more.

Reply:
<img width="367" alt="Screenshot 2025-04-15 at 2 21 37 PM" src="https://github.com/user-attachments/assets/03b6f3d9-8678-4cdd-abfd-1446df3b576a" />

I don't have a more defined grasp on why that's showing up now, except that it's now part of the textfield mod stack, so it makes sense that it's rendering now.